index.html

```diff
@@ -34,6 +34,187 @@ <h2>I'm a Software Developer</h2>
             <li><a href="https://twitter.com/anyarsencarta"><img src="./logos/Vector-M.png" alt="Linkedin"></a></li>
           </ul>
         </section>
+
+        <section class="extra">
+          <section class="works-centered">
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 for each card we could use the article tag instead of the section tag for the best practices of HTML & CSS.